manifests/init.pp

```diff
@@ -180,6 +185,7 @@
   Hash $config_defaults = {},
   String $os            = downcase($facts['kernel']),
   Optional[Variant[Stdlib::HTTPUrl, Stdlib::Unixpath, String[1]]] $external_url = undef,
+  Optional[Integer] $max_open_files = undef,
```

Comment:
@alexjfisher Could you please fix the indentation?

Reply:
I didn't think adding 45 extra whitespaces improved readability.

Reply:
Rebased (which added the `collect_scrape_jobs` line above mine)